packages/java-parser/src/parser.js

```diff
@@ -42,6 +42,18 @@ class JavaParser extends Parser {
       maxLookahead: 2,
       // ambiguities resolved by backtracking
       ignoredIssues: {
+        binaryExpression: {
+          OR: true
+        },
+        lambdaParameterType: {
+          OR: true
+        },
+        annotation: {
+          OR: true
+        },
+        localVariableType: {
+          OR: true
+        },
```

Comment:
These ones : binaryExpression, lambdaParameterType, annotation and localVariableTyp

Reply:
That is probably due to [changes in 4.5.0](https://sap.github.io/chevrotain/docs/changes/CHANGELOG.html#_4-5-0-5-25-2019) for better detecting of ambiguities related to token categories.

Make sure that when you add new ignores you are not hiding a bug.
Most Chevrotain parsers should not use these "ignoredIssues" but the Java Parser is different because of the use of backtracking. So make sure all the new "ignoredIssues" are actually  resolved by backtracking and not a true problem.

Reply:
annotation: we are using backtracking to resolve ambiguities
localVariableType and lambdaParameterType: same prefix of var for the var token but also unnannoted class type but this is a known issue about special handling of keywords https://github.com/jhipster/prettier-java/issues/113.
binaryExpression: "<" and ">" ambiguities but I think this is handled by looking token ahead.
Another ambiguities because of instanceof and BinaryOperator and BinaryOperator is a category of instanceof, I do not think this is an issue as we are trying to match the token instanceof first (and the following subrule are different I think)